sys/applemedia/Makefile.am

```diff
@@ -72,7 +75,7 @@ libgstapplemedia_la_LDFLAGS +=			\
 else
 
 libgstapplemedia_la_LDFLAGS +=			\
-	-Wl,-framework -Wl,Cocoa
-
+	-Wl,-framework -Wl,Cocoa \
+    -Wl,-framework -Wl,IOKit
```

Comment:
Note that we have to link with IOKit framework just to get the human readable Display name. It is linking with all the elements but only osxscreencapsrc needs it. Not ideal.